Steps4Impact/Networking/AKFCausesService.swift

```diff
@@ -127,6 +130,11 @@ class AKFCausesService: Service {
                                 completion: ServiceRequestCompletion? = nil) {
     shared.request(.delete, endpoint: .participant(fbId: fbid), completion: completion)
   }
+  
+  static func getParticipantSocial(fbid: String,
+                                   completion: ServiceRequestCompletion? = nil) {
+    shared.request(endpoint: .participantSocial(fbId: fbid), completion: completion)
+  }
```

Comment:
Is this still needed after the firebase switch?

Reply:
yep - only the current user's displayName and photoURL can be seen by the current user. Team Member/Leaderboard images are loaded using firebase's auth sdk from our api. Matching PR for that is here: https://github.com/AgaKhanFoundation/WCB/pull/148